include/ignition/sensors/AirPressureSensor.hh

```diff
@@ -67,7 +67,8 @@ namespace ignition
       /// \brief Update the sensor and generate data
       /// \param[in] _now The current time
       /// \return true if the update was successfull
-      public: virtual bool Update(const common::Time &_now) override;
+      public: virtual bool Update(
+        const std::chrono::steady_clock::time_point &_now) override;
```

Comment:
We've been using `std::chrono::steady_clock::duration` on `ign-gazebo` instead of `std::chrono::steady_clock::time_point` to represent specific simulation times. I don't remember exactly why it was done this way.

Reply:
This will required changes on:
  - ign-math: include `secNSecToDuration` function
  - ign-msgs : Change `convert` functions from `time_point` to `duration`
  - ign-gazebo and ign-sensor update to `std::chrono::steady_clock::duration`

This will required to make some releases. I have made the changes locally but I didn't push the changes yet. Should I go ahead?

Reply:
As discussed in the meeting, go for it! Thanks!